<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-43Q7UNA5" failures="0" tests="1" name="Default test" time="0.015" errors="0" timestamp="06 May 2021 07:47:29 GMT">
  <testcase classname="testbase.launchDriver" name="@BeforeSuite report" time="0.045">
    <failure type="java.lang.Error" message="Unresolved compilation problem: 
	VERSION_2_3_29 cannot be resolved or is not a field
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
VERSION_2_3_29 cannot be resolved or is not a field

at com.aventstack.extentreports.reporter.BasicFileReporter.start(BasicFileReporter.java:98)
at com.aventstack.extentreports.ReportObservable.register(ReportObservable.java:191)
at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658)
at com.aventstack.extentreports.ExtentReports.attachReporter(ExtentReports.java:82)
at testbase.launchDriver.report(launchDriver.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeSuite report -->
  <testcase classname="testbase.launchDriver" name="@BeforeTest dd" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeTest dd -->
  <testcase classname="testbase.launchDriver" name="@AfterMethod test" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod test -->
  <testcase classname="examplesselenium.Test_001" name="testabsdc" time="0.0">
    <skipped/>
  </testcase> <!-- testabsdc -->
</testsuite> <!-- Default test -->
